Formula/pyyaml.rb

```diff
@@ -0,0 +1,49 @@
+class Pyyaml < Formula
+  desc "YAML framework for Python"
+  homepage "https://pyyaml.org"
+  url "https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz"
+  sha256 "68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2"
+  license "MIT"
+
+  depends_on "cython" => :build
+  depends_on "python@3.10" => [:build, :test]
+  depends_on "python@3.8" => [:build, :test]
+  depends_on "python@3.9" => [:build, :test]
+  depends_on "libyaml"
+
+  def pythons
+    deps.select { |dep| dep.name.start_with?("python") }
+        .map(&:to_formula)
+        .sort_by(&:version)
+  end
+
+  def install
+    cythonize = Formula["cython"].bin/"cythonize"
+    system cythonize, "yaml/_yaml.pyx"
```

Comment:
Does this work even for multiple versions of Python? Huh.

Reply:
Yes. Cython's job is to translate the `*.pyx` file into a `*.c` file, which can then be compiled using a normal compiler. Once the `*.c` file exists, it can be processed multiple times -- one for each Python version.